src/System.Numerics/ref/System.Numerics.csproj

```diff
@@ -1,15 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <PackageTargetFramework>net46</PackageTargetFramework>
     <NuGetTargetMoniker>.NETFramework,Version=v4.6</NuGetTargetMoniker>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
-    <TargetingPackReference Include="System.Core" />
+    <Compile Include="System.Numerics.cs" />
+    <ProjectReference Include="..\..\mscorlib\ref\mscorlib.csproj">
+      <Project>{E9E2FEE9-3F81-4B59-A24C-0DBFC47C960C}</Project>
```

Comment:
Not that it matters much but the Project and Name metadata aren't actually needed for the ProjectReference's.